src/disco/metrics/metrics.xml

```diff
@@ -269,13 +269,8 @@ metric introduced.
     <counter name="HandshakesCreated" summary="Number of handshake flows created." />
     <counter name="HandshakeErrorAllocFail" summary="Number of handshakes dropped due to alloc fail." />
 
-    <counter name="StreamOpened" summary="Number of streams opened." />
```

Comment:
Should these be removed?

Reply:
Good point, we will need to extract the counterparts for these metrics from tpu_reasm

Reply:
Added back metrics for 'started', 'active', 'abandoned', and 'overrun' reassembly slots.